stacks/dashboards.yml

```diff
@@ -43,6 +43,7 @@ Parameters:
 
   DovetailAlbName: { Type: String }
   DovetailAlbFullName: { Type: String }
+  DovetailAppleApiBridgeApiId: { Type: String }
```

Comment:
This line should stand on its own, surrounded by whitespace. Each distinct app gets its own little section.